wps-common/src/main/java/au/org/aodn/aws/util/EmailService.java

```diff
@@ -113,4 +122,76 @@ public static String getCompletedJobEmailTemplate() {
     public static String getFailedJobEmailTemplate() {
         return String.format("%s/%s", WpsConfig.EMAIL_TEMPLATES_LOCATION, WpsConfig.FAILED_JOB_EMAIL_TEMPLATE_NAME);
     }
+
+    public static String formatRequestDetail(SubsetParameters subsetParameters, String collection) {
+
+        String details = "";
+
+        if (subsetParameters != null && collection != null) {
+
+            String spatialStr = EmailService.portalFormatSpatial(subsetParameters.getBbox());
+            String temporalStr = EmailService.portalFormatTemoral(subsetParameters.getTimeRange());
+            String depthStr = EmailService.portalFormatDepth(subsetParameters.getVerticalRange());
+
+            details = collection != null ? details.concat("Collection: " + collection + '\n') : details;
+            details = spatialStr != null ? details.concat(spatialStr + '\n') : details;
+            details = temporalStr != null ? details.concat(temporalStr + '\n') : details;
+            details = depthStr != null ? details.concat(depthStr + '\n') : details;
+        } else {
+            details = "(no request parameters are available)";
+        }
+
+        return details;
+    }
+
+    public static String portalFormatSpatial(LatLonRect bbox) {
+
+        if (bbox != null) {
+            String minLon = String.valueOf(bbox.getLonMin());
+            String minLat = String.valueOf(bbox.getLatMin());
+            String maxLon = String.valueOf(bbox.getLonMax());
+            String maxLat = String.valueOf(bbox.getLatMax());
+
+            String spatial = "";
+
+            if (minLon.equals(maxLon) && minLat.equals(maxLat)) {
+                spatial = spatial.concat("Timeseries at Lat/Lon: " + minLat + ',' + minLon);
+            } else {
+                spatial = spatial.concat("Spatial: " + minLon + ',' + minLat + ',' + maxLon + ',' + maxLat);
+            }
+
+            return spatial;
+        }
+
+        return null;
+    }
+
+    public static String formatDate(Date date) {
+
```

Comment:
Would be good to be consistent with new lines in the one class. I don't normally include them here.  Happy to put it to a vote though.

Reply:
Fixed.